presto-orc/src/test/java/com/facebook/presto/orc/TestTupleDomainOrcPredicate.java

```diff
@@ -148,6 +150,36 @@ private static ColumnStatistics doubleColumnStats(Long numberOfValues, Double mi
     }
 
     @Test
+    public void testFloat()
+            throws Exception
+    {
+        assertEquals(getDomain(FLOAT, 0, null), none(FLOAT));
+        assertEquals(getDomain(FLOAT, 10, null), all(FLOAT));
+
+        assertEquals(getDomain(FLOAT, 0, doubleColumnStats(null, null, null)), none(FLOAT));
+        assertEquals(getDomain(FLOAT, 0, doubleColumnStats(0L, null, null)), none(FLOAT));
+        assertEquals(getDomain(FLOAT, 0, doubleColumnStats(0L, (double) 42.24f, (double) 42.24f)), none(FLOAT));
+
+        assertEquals(getDomain(FLOAT, 10, doubleColumnStats(0L, null, null)), onlyNull(FLOAT));
+        assertEquals(getDomain(FLOAT, 10, doubleColumnStats(10L, null, null)), notNull(FLOAT));
+
+        assertEquals(getDomain(FLOAT, 10, doubleColumnStats(10L, (double) 42.24f, (double) 42.24f)), singleValue(FLOAT, rawFloat(42.24f)));
+
+        assertEquals(getDomain(FLOAT, 10, doubleColumnStats(10L, 3.3, (double) 42.24f)), create(ValueSet.ofRanges(range(FLOAT, rawFloat(3.3f), true, rawFloat(42.24f), true)), false));
+        assertEquals(getDomain(FLOAT, 10, doubleColumnStats(10L, null, (double) 42.24f)), create(ValueSet.ofRanges(lessThanOrEqual(FLOAT, rawFloat(42.24f))), false));
+        assertEquals(getDomain(FLOAT, 10, doubleColumnStats(10L, 3.3, null)), create(ValueSet.ofRanges(greaterThanOrEqual(FLOAT, rawFloat(3.3f))), false));
+
+        assertEquals(getDomain(FLOAT, 10, doubleColumnStats(5L, 3.3, (double) 42.24f)), create(ValueSet.ofRanges(range(FLOAT, rawFloat(3.3f), true, rawFloat(42.24f), true)), true));
+        assertEquals(getDomain(FLOAT, 10, doubleColumnStats(5L, null, (double) 42.24f)), create(ValueSet.ofRanges(lessThanOrEqual(FLOAT, rawFloat(42.24f))), true));
+        assertEquals(getDomain(FLOAT, 10, doubleColumnStats(5L, 3.3, null)), create(ValueSet.ofRanges(greaterThanOrEqual(FLOAT, rawFloat(3.3f))), true));
+    }
+
+    private long rawFloat(float value)
```

Comment:
Misleading method. I'd use static-imported `floatToRawIntBits` instead as we use in rest of code.